web/frontend/src/components/slam.vue

```diff
@@ -56,17 +56,14 @@ const fetchSLAMMap = (name: string): Promise<Uint8Array> => {
       const chunk = res.getPointCloudPcdChunk_asU8();
       chunks.push(chunk);
     });
-    getPointCloudMap.on('status', (status: { code: number, details: string, metadata: string }) => {
-      if (status.code !== 0) {
-        const error = {
-          message: status.details,
-          code: status.code,
-          metadata: status.metadata,
-        };
+    getPointCloudMap.on('status', (status?) => {
+      const { code, details, metadata } = status!;
+      if (code !== 0) {
+        const error = { message: details, code, metadata };
```

Comment:
@nicksanford the prior code was assuming that status is never `null` or `undefined` - I'm basically preserving that assumption with `status!` - let me know if this is okay.

also, I'm not quite sure why the linter wasn't complaining about this earlier...

Reply:
yep, this looks good to me

Reply:
Can you please test this change with fake slam to confirm that it doesn't break the SLAM UI? 

I'm happy to pair on this if it's not clear how to do so. Should take ~5 min.

Reply:
I ran the fake config and 2D and 3D maps seemed display fine while updating every second. Let me know if there's anything specific you'd like me to verify aside from the basic smoke test:

2D
![image](https://user-images.githubusercontent.com/7954103/232847924-a466ad7d-c693-4aa8-9c59-e2cc391cf81c.png)

3D
![image](https://user-images.githubusercontent.com/7954103/232848163-ea2be8e0-1857-444d-b5ec-766245a90eff.png)

Reply:
that looks good thanks!